src/rust/iced-x86-py/tests/instruction_test.py

```diff
@@ -1206,3 +1207,11 @@ def test_fpu_stack_increment_info():
 	assert info.increment == 1
 	assert not info.conditional
 	assert info.writes_top
+
+def test_pickle():
+	decoder = Decoder(64, b"\xC4\xE3\x49\x48\x10\x41" b"\xC4\xE3\x49\x48\x10\x42", ip=0x1234_5678_9ABC_DEF1)
```

Comment:
The 2nd string can be removed since it's just another instruction that never gets decoded.

When comparing the instructions with `==`, some fields are not compared (eg. the IP value), but the serializer/deserializer should always use every field. Change == to `assert instr.eq_all_bits(instr2)` to compare all bits.